### PR TITLE
Feature/vitest remove jest unit

### PR DESCRIPTION
**Pull request by MKaHead**
Remove everything arround unit test with Jest instead using vitest
- Remove all npm packages that was needed to run jest with unit test also polyfills.
- Remove integration tests from unit test
- Run unit test also in test script

- Remove obsolete postcss
- Remove obsolete autoprefixer

# Merge Checklist:
- [ ] Code selfreview has been performed ([Best Practices](https://github.com/axa-ch-webhub-cloud/pattern-library/blob/develop/CONTRIBUTION.md#best-practices)).
- [ ] Tests are sufficient.
- [ ] Documentation is up to date.
- [ ] Changelog is up to date.
- [ ] Typescript typings for properties are up to date.
- [ ] Designers approved changes or no approval needed.
- [ ] Dependencies to other components still work.

### CODE REVIEW

File: scripts/build/common.rollup.js [hunk 1]
```diff
@@ -28,11 +26,6 @@ module.exports.commonPlugins = [
     include: '/**/*.scss',
     includePaths: [path.resolve('node_modules')],
     prefix: globalSassImports,
-    processor: css =>
-      postcss({
-        plugins: [autoprefixer({ grid: 'autoplace' })],
-      })
-        .process(css, { from: undefined })
-        .then(result => result.css),
+    processor: css => css,
```
Comment (markus-walther) [inline scripts/build/common.rollup.js]: Question: do we still need a processor if it only does the identity transform?

**Review by markus-walther — DISMISSED** [review]
Marco,

1 question (could be answered/fixed, but not crucial), but looks good overall!

--Markus

File: scripts/build/common.rollup.js (hunk 1 above)
Comment (MKaHead) [reply scripts/build/common.rollup.js]: that a good catch, we don't need it!

**Review by selinagahlinger — APPROVED** [review]
Gemäss Markus alles in Ordnung.
Fix bzgl. processor sieht auch gut aus

**Pull request merged**